### PR TITLE
Change logo and remove api key 

### PR DESCRIPTION
**Pull request by MClimate**
#### Summary

...

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Compatibility: The changes are backwards compatible, they don't break existing deployments.
- [ ] Testing: The changes are tested.
- [ ] Documentation: Relevant documentation is added or updated.

### CODE REVIEW

File: mclimate.yml [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
+logo-url: https://cdn.shopify.com/s/files/1/0043/8874/9379/files/mclimate-logo-crop.jpg?v=1639751590
 info-url: https://mclimate.eu
 documentation-url: https://docs.mclimate.eu/mclimate-lorawan-devices/
 fields:
-  - id: api_key
-    name: API Key
-    description: API key used for authorization
-    secret: false
 format: json
 headers:
```
Comment (adriansmares) [inline mclimate.yml:9]: I think this should be `headers: []`

**Review by adriansmares — APPROVED** [review]

File: mclimate.yml (hunk 1 above)
Comment (nejraselimovic) [reply mclimate.yml:9]: I'll change this to `headers: {}` to be in line with the [documentation](https://www.thethingsindustries.com/docs/integrations/webhooks/webhook-templates/instantiation/#instantiation-of-header-values) and merge

**Comment by MClimate** [comment]
Hi,
We'd like to ask if we are doing something wrong as when we create an integration through the template, the join accept path is not checked. We wrote it in the YML file.
Thank you.

**Comment by adriansmares** [comment]
Yes, I see it now - the path uses `_` instead of `-`. You have `join_accept` instead of `join-accept`.

**Comment by nejraselimovic** [comment]
Yeah, I just submitted a PR to fix it https://github.com/TheThingsNetwork/lorawan-webhook-templates/pull/59

**Pull request merged**